access/types/forms.py

```diff
@@ -450,7 +454,15 @@ def grade_field(self, i, configuration):
                 else:
                     ok, hints, method = self.grade_checkbox(
                         self.row_options(configuration, row), value, hints)
-                if ok:
+                if self.exercise.get('feedback'):
+                    points += row.get("points", 0)
+                    max_points += row.get("points", 0)
+                elif "partial_points" in self.row_options(configuration, row):
+                    points += row.get("points", 0) * ok
```

Comment:
When you multiply with `ok`, shouldn't the result be casted to `int` since it becomes a float otherwise?